### DISCUSSION TITLE
Can I skip the DeFi section in the Foundry advanced course and start with airdrops and signatures, or is some knowledge from the previous section recommended?

### DISCUSSION THREAD

**Discussion by nikita-biichuk** [Cyfrin/foundry-full-course-cu#2983] [General]
Can I skip the DeFi section in the Foundry advanced course and start with airdrops and signatures, or is some knowledge from the previous section recommended?

**Comment by EngrPips** [comment]
Why do you want to skip it?

**Reply by nikita-biichuk** [reply]
seems it is hard therefore wanna leave for the end (to facilitate learning for myself maybe) and just interested

**Reply by EngrPips** [reply]
You can skip it and come back to it. It is not connected to the Airdrop course in any way.

**Reply by nikita-biichuk** [reply]
ok thanks!!

**Reply by EngrPips** [reply]
Anytime.

**Comment by heheboii11** [comment]
You may skip but I feel the advance Foundry course is a big hump you need to cross. If you complete it first, then it will be much easier to complete the remaining course. You will start to connect all the dots
